data/Makefile.am.inc

```diff
@@ -15,9 +15,17 @@ systemduserunit_DATA = xdg-desktop-portal-gtk.service
 
 CLEANFILES += $(systemduserunit_DATA)
 
+%.desktop: data/%.desktop.in
+	$(AM_V_GEN) msgfmt --desktop -d ../po --template $< -o $@
```

Comment:
There are no strings marked for translations at the moment in the desktop file, so maybe drop this and replace it with a block to replace `@libexec@` instead?
```
%.desktop: data/%.desktop.in
	$(AM_V_GEN) $(SED) -e "s|\@libexecdir\@|$(libexecdir)|" $< > $@
```
I think I'd move that snippet to a `$(do_edit)` var and reuse it for the desktop and the service files, though.

Or don't lose the plumbing for translations, but move that step to a `.in.in` file maybe?